streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -1250,6 +1253,20 @@ private Optional<String> removeStreamThread(final long timeoutMs) throws Timeout
         return Optional.empty();
     }
 
+    private int calculateMetricsRecordingLevel() {
+        final int recordingLevel;
+        final String recordingLevelString = applicationConfigs.getString(METRICS_RECORDING_LEVEL_CONFIG);
+        if (recordingLevelString.equals("INFO")) {
```

Comment:
Why not use a `switch` statement?

Reply:
I thought [Java 11 had switch expressions](https://docs.oracle.com/en/java/javase/13/language/switch-expressions.html) but it's Java 13+ so I was a little disappointed, but now looking at again the code will still benefit from using `switch`